src/ert/ensemble_evaluator/_builder/_legacy.py

```diff
@@ -165,14 +168,15 @@ async def _evaluate_inner(  # pylint: disable=too-many-branches
         is a function (or bound method) that only takes a CloudEvent as a positional
         argument.
         """
-        # Set up the timeout-mechanism
-        timeout_queue = asyncio.Queue()  # type: ignore
-        # Based on the experiment id the generator will
-        # give a function returning cloud event
-        event_creator = self.generate_event_creator(experiment_id=experiment_id)
-        on_timeout, send_timeout_future = self.setup_timeout_callback(
-            timeout_queue, cloudevent_unary_send, event_creator
-        )
+        if not isinstance(self._job_queue, Scheduler):
+            # Set up the timeout-mechanism
+            timeout_queue = asyncio.Queue()  # type: ignore
```

Comment:
This is a bit puzzling. Does not `on_timeout` require `timeout_queue`?
I am asking because we don't create it for scheduler:
```python
if not isinstance(self._job_queue, Scheduler):
            # Set up the timeout-mechanism
            timeout_queue = asyncio.Queue()  # type: ignore
```

Reply:
No, it is not needed in `_legacy.py` when we are doing the scheduler thing, since we move the entire thing down to `job.py` (and thereby always try to send the event even if `_legacy.py` has not asked for it)